Project1/calculator.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+#include <iostream>
+#include <string>
+
+enum class CalculatorException{
+	DIVIDE_BY_ZERO,
+	UNKNOWN_OPERATION
+};
+
+class Calculator {
+public:
+	/**
+	* calculates x + y
+	*/
+	double add(double x, double y);
```

Comment:
Does 'add' modifies an instance of Calculator?
If not - use the 'const' specifier to let people use calc.add on const instances of this class.